browser/resources/settings/brave_overrides/autofill_page.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import {RegisterPolymerTemplateModifications} from 'chrome://resources/brave/polymer_overriding.js'
+import {getTrustedHTML} from 'chrome://resources/js/static_types.js'
+
+import {loadTimeData} from '../i18n_setup.js'
+
+RegisterPolymerTemplateModifications({
+  'settings-autofill-page': (templateContent) => {
+    const addressesManagerButton =
+      templateContent.getElementById('addressesManagerButton')
+    if (!addressesManagerButton) {
+      console.error(`[Brave Settings Overrides] Couldn't find #addressesManagerButton`)
+    } else {
+      addressesManagerButton.insertAdjacentHTML(
```

Comment:
It seems like we can now use `html` function which would allow us to set the string attributes right inside instead of using setAttribute afterwards. For example, see https://github.com/brave/brave-core/blob/master/browser/resources/settings/brave_overrides/system_page.ts#L39

Reply:
looks much better, thanks!